x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/advanced_step/advanced_step_form.tsx

```diff
@@ -31,6 +33,42 @@ import { fetchExplainData } from '../shared';
 import { ContinueButton } from '../continue_button';
 import { OutlierHyperParameters } from './outlier_hyper_parameters';
 
+const defaultNumTopClassesOption: EuiComboBoxOptionOption = {
+  label: i18n.translate('xpack.ml.dataframe.analytics.create.allClassesLabel', {
+    defaultMessage: 'All classes',
+  }),
+  value: -1,
+};
+const zeroClassesMessage = i18n.translate(
+  'xpack.ml.dataframe.analytics.create.zeroClassesMessage',
+  {
+    defaultMessage:
+      'To evaluate the area under the curve of receiver operating characteristic (AUC ROC), use a non-zero value.',
+  }
+);
+const allClassesMessage = i18n.translate('xpack.ml.dataframe.analytics.create.allClassesMessage', {
+  defaultMessage:
+    'If you have a large number of classes there could be a significant effect on the size of your destination index.',
```

Comment:
Having the default setting give a red warning gives an odd user experience IMO. Could this be appended to the usual help text if they have 'All classes' selected, so that it appears in the usual grey font.

Reply:
Updated to be part of the help text when 'All classes' is selected - b6894dda6163f4f0dec5103aaa89066e83cd7840